index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   
+    <header id="nav-bar">
+        <a href="#"><h1>
+              Sonu
+            </h1></a>
+            <desktop id="header-desktop">
+            <h2 id="portfolio">Portoflio</h2>
+            <h3 id="about-header">About</h3>
+            <h4 id="contact-header">Contact</h4>
+            <img src="images/letter-icon.svg" id="letter-icon">

```

Comment:
This should have a display of hidden for mobile. Currently the `about` is visible. I suggest you remove them for now or set their display to none using `display:none`. As per the figma design the only two things visible are the logo and hamburger icon.
<img width="374" alt="Screenshot 2022-10-29 at 16 06 59" src="https://user-images.githubusercontent.com/48546992/198833497-34a9bf7c-1410-438d-9153-254771caaa5e.png">
This is you page currently
<img width="414" alt="Screenshot 2022-10-29 at 16 12 44" src="https://user-images.githubusercontent.com/48546992/198833526-f458355d-4cda-4722-9fbd-2d3cebea2a90.png">